src/main/java/org/team1540/robot2018/commands/climber/WinchOut.java

```diff
@@ -0,0 +1,27 @@
+package org.team1540.robot2018.commands.climber;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.team1540.robot2018.Robot;
+import org.team1540.robot2018.Tuning;
+
+public class WinchOut extends Command {
+
+  public WinchOut(){
+    requires(Robot.winch);
+  }
+
+  @Override
+  protected void initialize() {
+    Robot.winch.set(Tuning.winchOutSpeed);
+  }
+
+  @Override
+  protected boolean isFinished() {
+    return false; //Return true to stop the command
```

Comment:
Again, flag and accessor?

Reply:
`WinchOut` may actually be an issue as I believe that should be returning `true` since it's continuously called. However, AFAIK we're removing this entirely.